src/index.js

```diff
@@ -0,0 +1,174 @@
+// import modules
+const inquirer = require("inquirer");
+
+// import functions
+const {
+    selectAllFromDepartmentTable,
+    selectAllFromRoleTable,
+    selectAllFromEmployeeTable,
+} = require("./utils/queries");
+
+// import questions
+const {
+    askMenuQuestions,
+    askDepartmentQuestions,
+    roleQuestions,
+    employeeQuestions,
+    updateRoleQuestions,
+    updateManagerQuestions,
+    viewDepartmentQuestion,
+    deleteDepartmentQuestion,
+    deleteRoleQuestion,
+    deleteEmployeeQuestion,
+    calcUtilBudget,
+} = require("./utils/questions");
+
+// import DB
+const Db = require("./db/Db");
+
+// start app
+const start = async() => {
+    // db options
+    const db = new Db({
+        host: process.env.DB_HOST || "localhost",
+        user: process.env.DB_USER || "root",
+        password: process.env.DB_PASSWORD || "password123",
+        database: process.env.DB_NAME || "company_db",
+    });
+
+    // create connection to database
+    await db.start();
+
+    let active = true;
+
+    while (active) {
+        const { option } = await askMenuQuestions(inquirer);
+        console.log(option);
+
+        // department questions
+        if (option === "addDepartment") {
+            const { departmentName } = await askDepartmentQuestions(inquirer);
+            const query = `INSERT INTO department (name) VALUES ('${departmentName}');`;
+            await db.query(query);
+            console.log(`Added ${departmentName} into database!`);
+        }
```

Comment:
Just for consistency, try to keep things evenly spaced and leave a line between each block.